htdocs/projet/tasks/time.php

```diff
@@ -418,7 +418,8 @@
 		$tmptimespent = new Task($db);
 		$tmpproduct = new Product($db);
 		$fuser = new User($db);
-
+		$remiseproject = price2num(GETPOST('remiseproject', 'int'));
```

Comment:
Must use GETPOST('remiseprojet', 'alphanohtml') (see my answer to your question)